api/src/graphql/objects/user.rs

```diff
@@ -54,6 +62,7 @@ impl From<IdentityResponse<IdentityTrait>> for User {
             first_name: name.first_name,
             last_name: name.last_name,
             email,
+            profile_image: profile_image.unwrap_or_default(),
```

Comment:
The graphql type can be Option<String> so don't need to unwrap to an empty string. null more descriptive on it not being set.